settings/config.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please do not contaminate PRs with configuration files from your own environment.